dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/DataPointBaseInformation.kt

```diff
@@ -7,13 +7,11 @@ import org.dataland.datalandbackend.model.enums.data.QualityOptions
  * --- API model ---
  * Fields of a generic data point and its source
```

Comment:
Maybe write into the description, that this open class contains the minimal info for any datapoint on Dataland.

Reply:
Will do this now

Reply:
Done

Reply:
👍